### PR TITLE
add python-wechaty live coding demo

### PR DESCRIPTION
**Pull request by wj-Mcat**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

File: jekyll/_posts/2020-10-26-python-wechaty-live-coding.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - wechaty
+  - python-wechaty
+image: /assets/2020/python-wechaty/live-coding.png
+---
+
+## Python-Wechaty
+
+Wechaty 作为一个对话SDK，拥有适配多平台的优秀能力，同时还具备多语言的特性，今天我们将以一个简单的视频来介绍如何开始使用[Python-Wechaty](https://github.com/wechaty/python-wechaty)编写一个最简单的聊天机器人。
+
+{% include iframe.html src="https://player.bilibili.com/player.html?bvid=BV1dv411k75G" %}
```
Comment (huan) [inline jekyll/_posts/2020-10-26-python-wechaty-live-coding.md]: Please use YouTube as our default video service.

China-friendly links can be listed under the YouTube video.

Comment (wj-Mcat) [reply jekyll/_posts/2020-10-26-python-wechaty-live-coding.md]: Yes, I have change it to youtube channel.

Comment (huan) [reply jekyll/_posts/2020-10-26-python-wechaty-live-coding.md]: Could we use the `include` for the youtube video embedding?

I believe the youtube is supported by the include.

Comment (wj-Mcat) [reply jekyll/_posts/2020-10-26-python-wechaty-live-coding.md]: Done.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thanks for this great blog post, it will be our first Python Wechaty getting start live coding tutorial!

Please fix the conflict and then we will be able to merge it.

**Pull request merged**